src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -130,50 +118,53 @@ const workshops = [
 		elements: [
 			{
 				title: 'Hackathon 101',
-				author: 'Rajeshwari Jadhav',
-				description: `Start here for an idea of what Hack on the Hill 9 will offer as ACM Hack's
-				first hybrid hackathon. We'll also give you some tips on how to survive and thrive during
-				this year's HOTH! Have fun, good luck, and we believe in you!`,
-				youtube: 'https://youtu.be/GskV5qMOpkE',
+				author: 'Anan Wang',
+				description: `This workshop videos explains the basics of hackathons, which are events where
+				people work together to create technology projects. You'll learn what to expect and how to make
+				the most out of your experience. By the end of the video, you'll have a better understanding of
+				what hackathons are all about and how to get started with attending one. Happy hacking, handy
+				hackers!`,
+				youtube: 'https://youtu.be/p8ssQybQsd8',
 				readme: null,
-				slides: 'https://docs.google.com/presentation/d/1LNYFecHI-U5BGY9_OdqCTfPSFtq6wmDw9gwLld3tIVY/edit?usp=sharing'
+				slides: 'https://docs.google.com/presentation/d/1oln-vrDxRUNNjYx9DT616e8QW2x6HYdkriyty0N8Rkc/edit?usp=sharing'
 			},
 			{
 				title: 'Intro to Git',
-				author: 'Katelyn Yu',
-				description: `Welcome to the wonderous world of version control! In this
-				workshop, we'll introduce git and GitHub (two very different things) with a
-				quick guide on how to set them up and how to use them. We'll simulate creating
-				a project with git and GitHub with all the intricacies of terminal, branching,
-				merging, and merge conflicts. Curious as to what those are? Wake up your galaxy brain,
-				watch this workshop, and 'git gud' at git!`,
-				youtube: 'https://youtu.be/YWEOSz7Lo8U',
-				readme: 'https://github.com/uclaacm/hoth9-workshops/tree/main/intro-to-git',
-				slides: 'https://docs.google.com/presentation/d/11Gb0WyIzhGyTfzxS0bg_IhUuvTgwYifyVpzmMK8aOmg/edit?usp=sharing'
+				author: 'Jenna Wang',
+				description: `Git is a distributed version control system; it brings a local copy of the
```

Comment:
Would consider adding a brief explanation on what that means for those who are totally new
"Git is a distributed version control system-- it helps programmers collaborate on projects! It does this by..."

Reply:
I would but I don't wanna change what Jenna wrote too much 🤷‍♀️